util/src/main/java/com/graphicsfuzz/util/ToolHelper.java

```diff
@@ -25,13 +25,32 @@
 public class ToolHelper {
 
   public static ExecResult runValidatorOnShader(ExecHelper.RedirectType redirectType, File file)
-        throws IOException, InterruptedException {
+      throws IOException, InterruptedException {
+    return runValidatorOnShader(redirectType, file, false);
+  }
+
+  public static ExecResult runValidatorOnShader(ExecHelper.RedirectType redirectType, File file,
+                                                boolean vulkan)
```

Comment:
Change param to `isVulkan` and add JavaDoc (as per the comment at the start of the review for a similar method.)

Reply:
done